src/base/neural/owl_neural_graph_sig.ml

```diff
@@ -673,14 +673,16 @@ Load the weights from a file of the given name. Note that the weights and the
 name of their associated neurons are saved as key-value pairs in a hash table.
   *)
 
-  val get_subnetwork : ?make_inputs:string array -> node -> network
-  (**
-   Constructs a subnetwork of nodes on which ``node`` depends, replacing
-   nodes with names in ``make_inputs`` with input nodes. If ``make_inputs``
-   is empty or unspecified, the original input nodes are used.
-   
-   Note: the weights in the new network are the references of those in
-   the old one.
+  val get_subnetwork : ?copy:bool -> ?make_inputs:string array -> network -> string array -> network
+  (**
+   Constructs a subnetwork of nodes on which ``output_names`` depend,
```

Comment:
I think it would be useful to name the options here:
```
``get_subnetwork ?copy ?make_inputs network output_names`` constructs a subnetwork of nodes of ``network`` ...
```